pycbc/filter/qtransform.py

```diff
@@ -35,10 +35,72 @@
 from math import pi
 import numpy as np
 from pycbc.strain  import next_power_of_2
+from pycbc.types.timeseries import FrequencySeries, TimeSeries
+from numpy import fft as npfft
 
 __author__ = 'Hunter Gabbard <hunter.gabbard@ligo.org>'
 __credits__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 
+def qtransform(data, Q, f0, sampling, normalized):
```

Comment:
normalized should be a keyword argument, no? And set to either False or True by default.